pointcept/datasets/scannetpp.py

```diff
@@ -76,3 +76,28 @@ def get_data(self, idx):
         else:
             raise NotImplementedError
         return data_dict
+
+
+@DATASETS.register_module()
+class ScanNetPPTrainEvalSample(ScanNetPPDataset):
+    def __init__(
+        self,
+        nsamples=30,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.nsamples = nsamples
+
+    def __len__(self):
+        return self.nsamples
+
+
+@DATASETS.register_module()
+class ScanNetPPTestInstanceSegmentation(ScanNetPPDataset):
+    def prepare_train_data(self, idx):
```

Comment:
Overriding the `prepare_train_data` function , since the testing is not done in `test_mode` and the `name` field is needed to save the results for submission.